src/app/components/LiveLabel/Text/index.styles.ts

```diff
@@ -1,13 +1,32 @@
 import { css, Theme } from '@emotion/react';
 
 const styles = {
-  liveLabelText: ({ fontVariants, palette, spacings }: Theme) =>
+  liveLabelText: ({ fontVariants, spacings }: Theme) =>
     css({
-      color: palette.LIVE_DARK,
```

Comment:
I think we should still have a value for `palette.LIVE.DARK` somewhere? Otherwise the pulse and text won't match on some other pages 😭

Reply:
yeah 😭 i think thats one of the things that's causing the visual regression stuff out of the live page context , wooooops!